conductor/src/main/java/com/bluelinelabs/conductor/ChangeHandlerFrameLayout.kt

```diff
@@ -0,0 +1,39 @@
+package com.bluelinelabs.conductor
+
+import android.annotation.TargetApi
+import android.content.Context
+import android.os.Build
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import com.bluelinelabs.conductor.ControllerChangeHandler.ControllerChangeListener
+
+/**
+ * A FrameLayout implementation that can be used to block user interactions while
+ * [ControllerChangeHandler]s are performing changes. It is not required to use this
+ * ViewGroup, but it can be helpful.
+ */
+class ChangeHandlerFrameLayout : FrameLayout, ControllerChangeListener {
+
+    private var inProgressTransactionCount = 0
+
+    constructor(context: Context) : super(context)
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
```

Comment:
Perhaps it's better to use 

```kotlin
class ChangeHandlerFrameLayout @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null)
```
instead of manually specifying each constructor?

Reply:
I generally strongly dislike JvmOverloads for View constructors.
Different constructors might have different behavior (and if they dont have right now that might change in a future update).

Reply:
I see, ok then.